README.md

```diff
@@ -1 +1,7 @@
-# .github
\ No newline at end of file
+# OpenJS Foundation GitHub Configuration
```

Comment:
Dont need this file. Just duplicate the readme in profile/readme here and its ok.